app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -184,14 +186,22 @@ fun OtherProfileScreenContent(
                     color = MaterialTheme.wireColorScheme.background
                 ) {
                     Box(modifier = Modifier.padding(all = dimensions().spacing16x)) {
-                        OtherUserConnectionActionButton(
-                            state.connectionStatus,
-                            onSendConnectionRequest,
-                            onOpenConversation,
-                            onCancelConnectionRequest,
-                            acceptConnectionRequest,
-                            ignoreConnectionRequest
-                        )
+                        // TODO show open conversation button for bots after AR-2135
+                        if (state.membership == Membership.Service) {
+//                            WirePrimaryButton(
+//                                text = stringResource(R.string.label_open_conversation),
+//                                onClick = onOpenConversation,
+//                            )
```

Comment:
I would be better if we just remove this commented code :D

Reply:
Sure